dilib/container.py

```diff
@@ -51,23 +54,31 @@ def config(self) -> TC:
         # Cast because ConfigProxy[TC] will act like TC
         return cast(TC, ConfigProxy(self, self._config))
 
+    # noinspection PyProtectedMember
```

Comment:
What do you think about making `config._keys` part of the public interface, so we don't need to suppress these PyCharm warnings?

Reply:
I'm not sure--on the one hand, I've thought about adding "meta" helpers that expose the delayed spec structure, but on the other hand, I'm not sure what the use case is for that, and I wanted to keep the public interface as small as possible. I thought of these internal `Config` indices as specifically designed for the "friend" `Container` class to use.